pycodestyle.py

```diff
@@ -275,6 +276,9 @@ def maximum_line_length(physical_line, max_line_length, multiline, noqa):
     line = physical_line.rstrip()
     length = len(line)
     if length > max_line_length and not noqa:
+        # Special case: ignore long shebang lines.
+        if line_number == 1 and length > 1 and line[:2] == '#!':
```

Comment:
`length > 1` seems redundant to me, since we already know `length > max_line_length` from L280.

Also, what do you think of doing `line.startswith('#!')` instead of `line[:2] == '#!'`?  It's more characters, but seems *slightly* more readable to me.